packages/concerto-core/lib/rootmodel.js

```diff
@@ -32,7 +32,10 @@ function getRootModel(versioned) {
     abstract concept Participant identified {}
     abstract concept Transaction {}
     abstract concept Event {}
-    `;
+    abstract concept Decorator {}
+    concept DotNetNamespace extends Decorator {
+       o String namespace
+    }`;
```

Comment:
Adding declarations to the root model without bumping the version?

Reply:
Moved this to a new ns.